### PR TITLE
Fixes typos

### PR DESCRIPTION
**Pull request by icanzilb**
As per the title — it fixes a number of typos in the source code

### CODE REVIEW

File: Sources/STTextView/Utility/TypingTextUndo.swift [hunk 1]
```diff
@@ -5,5 +5,5 @@ import AppKit
 
 struct TypingTextUndo {
     let textRange: NSTextRange
-    let attribugedString: NSAttributedString?
+    let attribuggedString: NSAttributedString?
```
Comment (krzyzanowskim) [inline Sources/STTextView/Utility/TypingTextUndo.swift:8]: That meant to be `attributedString`

Comment (icanzilb) [reply Sources/STTextView/Utility/TypingTextUndo.swift:8]: I put a lot of effort into this, would you not reconsider?

Comment (icanzilb) [reply Sources/STTextView/Utility/TypingTextUndo.swift:8]: well ok, you convinced me ... changing this

**Review by krzyzanowskim — APPROVED** [review]
thank you!

**Pull request merged**